notebooks/parameterize_script/util/argument_parsing.py

```diff
@@ -55,3 +55,34 @@ def get_script_name(args: Optional[Sequence[str]] = None) -> ScriptName:
         parser.exit()
 
     return ScriptName.from_string(known_args.script_name)
+
+def get_log_level(args: Optional[Sequence[str]] = None) -> str:
+    """
+    Parses the log level option from the program arguments.
+    INFO is the default log level
+    This will exit if the log level in an invalid choice.
+    Args:
+        args (Optional[Sequence[str]]): The program arguments.
+            By default, command line arguments will be used.
+    Returns:
+        str: The value of the --log_level argument
+    """
+
+    parser: argparse.ArgumentParser = argparse.ArgumentParser(
+        add_help=False
+    )
+
+    parser.add_argument(
```

Comment:
I find this a bit confusing. We've already added our arguments and parsed them in `script_instance.parse_args()`, so it doesn't feel right to be adding more arguments and re-parsing here.
Should `MySqlToSpannerScript.parse_args` add global/common arguments instead?
Something like:
```
def parse_args(args: Optional[Sequence[str]] = None) -> Dict[str, Any]:
    parser = argparse.ArgumentParser()
    add_common_args(parser)
    parser.add_argument(
            f'--{constants.OUTPUT_NOTEBOOK_ARG}',
...
```

The the line in `run_script()` would be:
```
logging.basicConfig(level=args.log_level, format="%(message)s")
```

This is just an idea, feel free to reject it. It just seems odd to keep re-parsing the arguments. Same goes for `get_script_name()`, that would be added and validated parse_args too.

Reply:
common_args() is a nice idea, will implement that